src/main/resources/com/puppycrawl/tools/checkstyle/grammar/java.g

```diff
@@ -1693,7 +1757,6 @@ options {
     {
         mTreatEnumAsKeyword = aTreatAsKeyword;
     }
-
```

Comment:
This blank line has a pair on line 1668 (1732). Please remove both of them or none of them.

Reply:
Fixed.